docs/configuration/configuration.md

```diff
@@ -3529,6 +3533,10 @@ ovhcloud_sd_configs:
 puppetdb_sd_configs:
   [ - <puppetdb_sd_config> ... ]
 
+# List of Tailscale service discovery configurations.
+tailscale_sd_configs:
```

Comment:
Looks like this SD list is mostly (but not fully) alphabetically sorted at the moment, so maybe let's put it below `serverset_sd_configs`?

Reply:
I've moved it here and made sure it's alphabetical in all the places.